test/addons/e10s-remote/main.js

```diff
@@ -0,0 +1,581 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+const LOCAL_URI = "about:robots";
+const REMOTE_URI = "about:home";
+const REMOTE_MODULE = "./remote-module";
+
+const { Loader } = require('sdk/test/loader');
+const { getTabs, openTab, closeTab, setTabURL, getBrowserForTab, getURI } = require('sdk/tabs/utils');
+const { getMostRecentBrowserWindow } = require('sdk/window/utils');
+const { cleanUI } = require("sdk/test/utils");
+const { setTimeout } = require("sdk/timers");
+
+const { Cu } = require('chrome');
+const { async } = Cu.import('resource://gre/modules/Task.jsm', {}).Task;
```

Comment:
nit we can just require this `const { Task: async } = require('resource://gre/modules/Task.jsm');`